### ISSUE TITLE
[Adaptive Cards - Stock Update]: Screen Reader does not announce the information about the stocks going up on Stock Update pane.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9017]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: Dev (OS Build 27695.1000)
Browser Version: Version 130.0.2849.1 (Official build) dev (64-bit)
Screen Reader: Narrator

**Pre-Requisite:**
Turn on Narrator and turn on scan mode using caps+ spacebar key

**Repro steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to the 'Sample' option and press enter key to activate it.
3. Press tab key and navigate to the 'Stock Update' link and press enter key to activate it.
4. Press tab key and navigate the card and press arrow key and navigate to the stock update option and observe the issue.

**Actual Result:**
Screen Reader does not announces the information about the stocks going up on Stock Update pane. Screen Reader is announcing, "2.69 USD 2.13%".

**Expected Result:**
Screen Reader should announce the complete information about the content present on Stock Update pane as, "2.69 USD Stocks are rising by 2.13%" or "Stock increment by 2.13%".

**Note:**
Same issue is repro with NVDA.

**User Experience:**
Users who depends on AI might get confused if screen reader is not announcing the complete information about the content present on Stock Update pane.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Screen Reader does not announce the information about the stocks going up on Stock Update pane](https://github.com/user-attachments/assets/9779119b-00ed-4836-b9ae-3c82a59293ac)

https://github.com/user-attachments/assets/b6c1bf95-ebb4-44b9-8759-b1594c829d96

### Card JSON

```JSON
NA
```
